tests/CMakeLists.txt

```diff
@@ -94,6 +94,9 @@ set_property(
 endif()
 target_compile_definitions(bpftrace_test PRIVATE HAVE_SEND_SIGNAL)
 
+if(HAVE_GET_FD_PATH)
+  target_compile_definitions(bpftrace_test PRIVATE HAVE_GET_FD_PATH)
```

Comment:
For send_signal I've always defined this(unconditionally), and manually defined BPF_FUNC_send_signal to always have the tests work, as we can do semantic and codegen tests without kernel support. You'll will have to do the same thing here.

Reply:
I see, sounds good